lms/djangoapps/progress/signals.py

```diff
@@ -67,10 +71,12 @@ def save_history(sender, instance, **kwargs):  # pylint: disable=no-self-argumen
     """
     Event hook for creating progress entry copies
     """
+    # since instance.completions return F() ExpressionNode we have to pull completions from db
```

Comment:
Can you add a URL to a doc in this comment for people (eg, me) to read up on F and ExpressionNode?